### ISSUE TITLE
Add documentation/example showing how to retrieve the model with the best performance

### ISSUE THREAD

**Issue by 610v4nn1** [awslabs/syne-tune#296]
It would be useful to have an example showing how to retrieve the trained model providing the best performance.

**Comment by geoalgo** [comment]
Hi @610v4nn1 ,

Could you precise which backend you are using? 
Retrieving the model could mean a few different things depending whether you are using the local or SageMaker backend. 

First note that if you do not checkpoint in your training script, then you wont be able to retrieve the model (Syne Tune cannot know how to save your model) and also that the best trial-id is written at the end of the tuning with a line like this one:
```mean_loss: best -7.754014598540146 for trial-id 4```

If you activated checkpointing, the checkpoint files will be written to disk/S3 for local/sagemaker backends as explained [here](https://github.com/awslabs/syne-tune/blob/main/docs/faq.md#tuning-output-location) and you will be able to load the model for the best trial there.

**Comment by geoalgo** [comment]
True, one thing is that this would require to explain how to deploy a model in SageMaker which requires a few explanations. Feel free to send a PR if you have an idea on how to explain it well.

**Comment by 610v4nn1** [comment]
It would be good to label the issue and leave it open such that we can get back to this.
It's much easier to forget about adding the example when the issue is closed.
